src/app/theme/SplashScreenService.ts

```diff
@@ -65,14 +68,18 @@ export const showSplashScreen = async (
   setTimeoutForSplashScreen(bootsplashName)
 
   try {
-    await RNBootSplash.show({ fade: true, bootsplashName })
-    return splashScreenLogger.info(`Splash screen shown "${bootsplashName}"`)
+    const result = await RNBootSplash.show({ fade: true, bootsplashName })
+    splashScreenLogger.info(
+      `Splash screen shown "${bootsplashName}" (${result.toString()})`
```

Comment:
seems like there are typing issues detected by github ci here

Reply:
This is because [of](https://github.com/cozy/react-native-bootsplash/pull/4) is not merged yet